worlds/generic/docs/setup_en.md

```diff
@@ -1,18 +1,21 @@
 # Archipelago Setup Guide
 
-This guide is intended to provide an overview of how to install, set up, and run the Archipelago multiworld software.
-This guide should take about 5 minutes to read.
+This guide is intended to provide an overview of how to install, set up, and run the Archipelago multiworld software, as
+well as how to generate and host multiworlds, and how to connect to the multiworld after hosting has begun. This is a
+general overview; for more specific steps, reference the relevant game's [setup guide](/tutorial).
 
 ## Installing the Archipelago software
```

Comment:
The final paragraph of this section could be reworded to avoid the game/games singular/plural verb tense changes. "If the game/games you plan to play are available here go ahead and install these as well." could be changed to "If the game you plan to play is available here go ahead and install its client as well." There should also be a comma in the sentence "If the game you choose to play is supported by Archipelago but not listed in the installation check the setup guide for that game." between "installation" and "check"